README.md

```diff
@@ -5,3 +5,313 @@ qgepplugin
 This plugin is part of the QGEP Wastewater project for QGIS.
 
 [![Build Status](https://travis-ci.org/QGEP/qgepplugin.svg?branch=master)](https://travis-ci.org/QGEP/qgepplugin)
+
+# QGEP-SWMM
+
+## Installation
+
+### Install QGEP plugin 
+
+* Download the plugin: https://github.com/iggroup/qgepplugin/tree/swmm/src
+* Save in the QGIS plugin path in a folder named `qgepplugin`.
+
+### SWMM Installation
+* Download and install [SWMM](https://www.epa.gov/water-research/storm-water-management-model-swmm)
+
+* In QGIS `Preferences > Processing > Processing providers > QGEP > SWMM exectutable` set the path to the swmm `.exe` i.e. `C:/Program Files (x86)/EPA SWMM 5.1.013/swmm5.exe` 
+
+### Installation psycopg2
+The script requires `psycopg2` to connect to the database. It is installed by default with QGIS, it is not necessary to install it if you use the QGEP plugin.
```

Comment:
I'm a bit confused about the difference between "the script" and "the QGEP plugin"

Reply:
One could want to use the QgepSwmm.py class without QGIS. In this scenario, he would have to install psycopg2. Anyway, this scenario, is unlikely. I deleted this part.